dask_ml/model_selection/_incremental.py

```diff
@@ -837,9 +834,40 @@ def _get_params(self):
         if self.n_initial_parameters == "grid":
             return ParameterGrid(self.parameters)
         else:
-            return ParameterSampler(self.parameters, self.n_initial_parameters)
+            return ParameterSampler(
+                self.parameters,
+                self.n_initial_parameters,
+                random_state=self.random_state,
+            )
 
     def _additional_calls(self, info):
+        calls = {k: v[-1]["partial_fit_calls"] for k, v in info.items()}
+        if all(c == 1 for c in calls.values()):
+            # this condition happens at the beginning
+            #
+            # _to_reach records the number of partial_fit calls that need to be
+            # be made on each model to reach the number _adapt specifies.
+            # It only matters for when patience is set -- otherwise,
+            # the calls specified by _adapt are scheduled.  It's a class
+            # variable because this state needs to be stored somewhere.
```

Comment:
Can you double check the "stored somewhere" statement? I don't see any uses of it outside of `_additional_calls`, in which case it can be a regular variable.

Reply:
I've expanded on the documentation to clarify this:

> _to_reach records the number of partial_fit calls that need to be
be made on each model to reach the number _adapt specifies.
It only matters for when patience is set -- otherwise,
the calls specified by _adapt are scheduled.
>                                                                  
> It's a class variable because this variable needs to be stored
somewhere.  Nothing is known about _adapt, so _to_reach needs
to be persisted across calls to _additional_calls. It can no
be calculated every call because _adapt is a black box.